gpt4all-chat/build_and_run.md

```diff
@@ -24,6 +24,11 @@ On Ubuntu 23.04, this looks like:
 sudo apt install build-essential qt6-base-dev qt6-declarative-dev qt6-wayland-dev qt6-svg-dev qt6-httpserver-dev qt6-webengine-dev libqt6core5compat6 qml6-module-qt5compat-graphicaleffects libqt6shadertools6 qtcreator cmake ninja-build
 ```
 
+On Debian 12, this looks like:
+```
+sudo apt install build-essential qt6-base-dev qt6-declarative-dev qt6-wayland-dev qt6-wayland qt6-svg-dev qt6-httpserver-dev qt6-webengine-dev qt6-websockets-dev libqt6core5compat6 qml6-module-qt5compat-graphicaleffects libqt6shadertools6 qtcreator cmake ninja-build

```

Comment:
These packages work the same way on Debian as they do on Ubuntu, so if anything this should be a modification to the Ubuntu command. But are you sure this is right?

We only require WebSockets as a dependency of HttpServer, and libqt6websockets6 will be installed if you install qt6-httpserver-dev. We do not reference WebSockets in our build scripts or #includes.

And in theory you shouldn't need qt6-wayland just to compile GPT4All. Maybe you need it to run GPT4All on native Wayland, but that would be an optional dependency and not a required one - GPT4All can also run on X11.